docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -321,111 +437,67 @@ To decode a JPEG, PNG, GIF, or BMP image:
 
 To encode a raw image:
 
-1. Create an encoding handle using the `image_util_encode_create()` function:
+1. Create an encoding handle using `image_util_encode_create()`:
 
    ```
    image_util_type_e encoder_type = IMAGE_UTIL_JPEG;
    image_util_encode_h encode_h = NULL;
    ret = image_util_encode_create(encoder_type, &encode_h);
    ```
 
-2. Set the image to the input buffer using the `image_util_encode_set_input_buffer()` function:
-
-   ```
-   ret = image_util_encode_set_input_buffer(encode_h, buffer);
-   ```
-
-3. Set the width, height, and color space of the input buffer using the `image_util_encode_set_resolution()` and `image_util_encode_set_colorspace()` functions:
+2. Additionally, you can set the JPEG quality or PNG compression using `image_util_encode_set_quality()` or `image_util_encode_set_png_compression()`:
 
    ```
-   ret = image_util_encode_set_resolution(encode_h, width, height);
-   ret = image_util_encode_set_colorspace(encode_h, IMAGE_UTIL_COLORSPACE_RGBA8888);
+   ret = image_util_encode_set_jpeg_downscale(decode_h, IMAGE_UTIL_DOWNSCALE_1_1);
    ```
 
    > **Note**
    >
-   > - Because of encoder limitations, color space setting is only supported for encoding JPEG images.
-   > - The default color space is `IMAGE_UTIL_COLORSPACE_RGBA8888`. PNG, GIF and BMP images are encoded with `IMAGE_UTIL_COLORSPACE_RGBA8888`.
-
-4. To save the encoded image, set the output path or buffer using the `image_util_encode_set_output_path()` or `image_util_encode_set_output_buffer()` function:
-
-   ```
-   ret = image_util_encode_set_output_path(encode_h, path);
-   ```
+   > Due to the encoder limitations, the quality setting is only supported for encoding the JPEG images, and compression is only supported for the PNG images. The default JPEG quality is 75. The default PNG compression is `IMAGE_UTIL_PNG_COMPRESSION_6`.
 
-5. Optionally, set the JPEG quality or PNG compression using the `image_util_encode_set_quality()` or `image_util_encode_set_png_compression()` function:
+3. Execute the encoding using `image_util_encode_run_to_file()` or `image_util_encode_run_to_buffer()`:
 
    ```
-   ret = image_util_encode_set_jpeg_downscale(decode_h, IMAGE_UTIL_DOWNSCALE_1_1);
+   ret = image_util_encode_run(encode_h, decoded_image, file);
```

Comment:
`image_util_encode_run_to_file()` or `image_util_encode_run_to_buffer()` in description, and the sample code is `image_util_encode_run()`.
I can see `image_util_encode_run()` in the API file anyway, however would you please check?

Reply:
`image_util_encode_run` should be changed to `image_util_encode_run_to_file in the sample code.
I will replace the sample code.